contracts/actions/convex/helpers/ConvexHelper.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "../../../interfaces/convex/IConvexToken.sol";
+import "../../../interfaces/convex/IConvexToken.sol";
+import "../../../interfaces/convex/IBaseRewardPool.sol";
+import "../../../interfaces/convex/IRewardPool.sol";
+import "../../../utils/SafeMath.sol";
+import "../../../utils/TokenUtils.sol";
+
+import "./MainnetConvexAddresses.sol";
+
+contract ConvexHelper is MainnetConvexAddresses {
+    using SafeMath for uint256;
+    using TokenUtils for address;
+
+    uint256 internal immutable CVX_REDUCTION_PER_CLIFF;
+    uint256 internal immutable CVX_TOTAL_CLIFFS;
+    uint256 internal immutable CVX_MAX_SUPPLY;
+
+    struct Reward {
+        address token;
+        uint256 earned;
+    }
+
+    enum DepositOption {
+        WRAP,
+        STAKE,
+        WRAP_AND_STAKE
+    }
+
+    enum WithdrawOption {
+        UNWRAP,
+        UNSTAKE,
+        UNSTAKE_AND_UNWRAP
+    }
+
+    constructor() {
```

Comment:
All constants that can't change no need to add constructor and initialize it on the fly